.github/ISSUE_TEMPLATE/content-or-feature-suggestion.yml

```
@@ -1,68 +1,31 @@
-name: New content or feature requests
+name: New content or feature suggestion
 description: Proposals for new content or features on MDN Web Docs
-title: A short title that begins with a verb
-labels: ["content suggestion", "feature suggestion", "proposed"]
+title: Enter your proposal here
+labels: ["suggestion", "proposed", "needs triage"]
```

>Not sure if we need both suggestion and proposed 

Agree with this. Shall we do these two:

```suggestion
labels: ["proposal", "needs triage"]
```

---

Side-note, content has `needs-triage` instead of `needs triage`, we should make these the same to help with future us when we want to search for these labels across repos